template/docs/conf.py.jinja

```diff
@@ -111,10 +112,8 @@ master_doc = 'index'
 # built documents.
 #
 
-# The short X.Y version.
-version = {{ projectname.removeprefix(namespace_package) }}.__version__
-# The full version, including alpha/beta/rc tags.
-release = {{ projectname.removeprefix(namespace_package) }}.__version__
+release = get_version("{{projectname}}")
```

Comment:
Can you tell me how this works? Does `get_version` query the installed package, even if `projectname` is not importable (in case of namespace packages)?

Reply:
👍 
For future reference, I also found this explanation, based on your link: https://packaging.python.org/en/latest/discussions/distribution-package-vs-import-package/#distribution-package-vs-import-package